src/libstd/macros.rs

```diff
@@ -460,9 +460,12 @@ pub mod builtin {
     /// With [`column!`] and [`file!`], these macros provide debugging information for
     /// developers about the location within the source.
     ///
-    /// The expanded expression has type `u32`, and the returned line is not
-    /// the invocation of the `line!()` macro itself, but rather the first macro
-    /// invocation leading up to the invocation of the `line!()` macro.
+    /// The expanded expression has type `u32` and is 1-based, so the first line
+    /// in each file evaluates to 1, the second to 2, etc. This is consistent
+    /// with error messages by common compilers or popular editors.
+    /// The returned column is not the invocation of the `line!` macro itself,
```

Comment:
This should say the returned \*line\*.

Reply:
Since we are touching this line anyway, I find "not the invocation of the \`line!\` macro itself" a bit misleading, because it is totally possible and expected for the return value to be the line of the invocation of the line macro itself when not being called from another macro. It may be clearer as "not necessarily the line of the \`line!\` invocation itself".

(Okay to ignore this if you want, since it was that way before.)